app/views/step_by_step_pages/show/_required_actions.html.erb

```diff
@@ -45,9 +45,15 @@
 <% if @step_by_step_page.status.submitted_for_2i? %>
   <%= render "govuk_publishing_components/components/inset_text" do %>
     <p class="govuk-body">Not yet claimed for 2i</p>
+    <% if current_user.permissions.include?("2i reviewer") %>
+      <a href="/step-by-step-pages/<%=@step_by_step_page.id%>/guidance">Guidance</a>
```

Comment:
There's a `link_to` helper method that will create the relevant HTML for you and also include the correct path, e.g.
```
<%= link_to "Guidance", step_by_step_page_guidance_path(@step_by_step_page) %>
```